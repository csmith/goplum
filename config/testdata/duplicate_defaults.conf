defaults {
    alerts = ["text", "foo", "bar", ]
	good_threshold = 3
	interval = 1m30s
}

alert twilio.message "text" {
    sid = "xxx"
    token = "yyy"
    from = "+44 ..."
    to = "+44 ...."
}

check http.get "chameth.com" {
    url = "https://chameth.com/"
    content = "Personal Homepage of Chris Smith"
}

plugin http {
    foo = "bar"
}

defaults {
    alerts = []
}
