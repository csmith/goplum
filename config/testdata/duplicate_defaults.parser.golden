"duplicate defaults block declared at line 23"